src/games/weakschur/weakschur_state.h

```diff
@@ -124,7 +124,7 @@ void weakschur::State<NBSUBSETS, MAXNUMBER>::ApplyAction(
 
   const int channelSize = NBSUBSETS * MAXNUMBER;
 
-  auto feature = [channelSize, this](int c, int subset, int number) -> float& {
```

Comment:
how was this ever working?

Reply:
That works just fine, capturing this variable. It's just unnecessary apparently according to Clang.